### ISSUE TITLE
Add support for Packery 3.x

### ISSUE THREAD

**Issue by joshuahiggins** [SunGard-Labs/angular-packery#22]
It seems the recent update to Packery 3.0 has caused some issues with this repo. As a temporary workaround, the bower should be updated to only pull 2.x versions as we add support.

**Comment by leads** [comment]
Yep, getting errors when trying to bower install this into a Angular 1.3 version
ECONFLICT Unable to find suitable version for angular

**Comment by pmcelhaney** [comment]
Packery is up to 2.0 now. :)
